CONTRIBUTING_DOCS.md

```diff
@@ -143,7 +136,7 @@ You can also create custom callouts using the `call-out` shortcode `{{< call-out
 {{<call-out "important side-callout" "JWT file required for upgrade" "fa fa-exclamation-triangle">}}
 ```
 
-By default, all custom callouts are included inline, unless you add `side-callout` which places the callout to the right of the content.
+By default, all custom callouts are displayed inline, unless you add `side-callout` which places the callout to the right of the content.
```

Comment:
Is there guidance on when to use in-line vs. out-dented callouts?

Reply:
No, and I actually have no knowledge of any instances when this has been used.

Just thought I'd change the word used since "include" has a certain semantic weight with our Hugo shortcode.